tools/db_crashtest.py

```diff
@@ -1000,11 +1000,11 @@ def finalize_and_sanitize(src_params):
             dest_params["exclude_wal_from_write_fault_injection"] = 1
             dest_params["metadata_write_fault_one_in"] = 0
     # Disabling block align if mixed manager is neing used
-    if dest_params.get("compression_manager") == "mixed":
+    if dest_params.get("compression_manager") != "none":
```

Comment:
why can't we just do "if dest_params.get("compression_manager") == "mixed":"? I don't think your auto-skip need special treatment.